package.json

```diff
@@ -15,15 +19,14 @@
     "*.d.ts.map"
   ],
   "scripts": {
+    "prefetch": "mkdirp ontologies",
     "fetch": "ts-node fetch.ts",
     "lint": "eslint . --ext .js,.ts",
     "pretest": "npm run lint",
     "test": "jest",
-    "build:node": "tsc --target es2017 --outDir lib/node",
-    "build:modern": "tsc --module esnext --outDir lib/es",
-    "build:types": "tsc --outDir . --declaration --declarationMap --emitDeclarationOnly",
-    "build:es-loader": "tsc src/es/loadDataset.ts --target esnext --module esnext --esModuleInterop --moduleResolution node --outDir lib/es",
-    "build": "rimraf lib; npm run build:node; npm run build:modern; npm run build:es-loader; npm run build:types",
+    "build:node": "tsc",
+    "build:modern": "babel src --out-dir lib --extensions .ts --out-file-extension .mjs --ignore *.test.js --ignore src/loadDataset/index.ts",
```

Comment:
`src/loadDataset/index.ts` ignored from babel build because the actual `loadDataset/index.mjs` is a completely different module